src/wireviz/DataClasses.py

```diff
@@ -3,10 +3,46 @@
 
 from typing import Optional, List, Any, Union
 from dataclasses import dataclass, field
-from wireviz.wv_helper import int2tuple
+from pathlib import Path
+from wireviz.wv_helper import int2tuple, aspect_ratio
 from wireviz import wv_colors
 
 
+@dataclass
+class Image:
+    gv_dir: Path # Directory of .gv file injected as context during parsing
```

Comment:
Maybe the type should be `InitVar[Path]` because it is [only used during initialization](https://docs.python.org/3/library/dataclasses.html#init-only-variables) in the `__post_init__()` function?